<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_win">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Выиграл игрок:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ura">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Поздравляем!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_real_cards">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть реальные карты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_menu">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>В главное меню</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_real_sum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>С суммой реальных карт:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_real_sum">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pic">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_back_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lbl_back_pic</zorder>
   <zorder>lbl_win</zorder>
   <zorder>lbl_ura</zorder>
   <zorder>b_real_cards</zorder>
   <zorder>b_menu</zorder>
   <zorder>lbl_real_sum</zorder>
   <zorder>lcd_real_sum</zorder>
   <zorder>lbl_name</zorder>
   <zorder>lbl_pic</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
